cli/borges/version.go

```diff
@@ -8,9 +8,30 @@ const (
 	versionCmdLongDesc  = versionCmdShortDesc
 )
 
-type versionCmd struct{}
+var versionCommand = &versionCmd{simpleCommand: newSimpleCommand(
+	versionCmdName,
+	versionCmdShortDesc,
+	versionCmdLongDesc,
+)}
+
+type versionCmd struct {
+	simpleCommand
+}
 
 func (c *versionCmd) Execute(args []string) error {
-	fmt.Printf("%s - %s (build %s)\n", name, version, build)
+	println("VERSION")
+	fmt.Printf("%s - %s (build %s)\n", borgesName, version, build)
 	return nil
 }
+
+func init() {
```

Comment:
i'd prefer initializing the commands in the main as it was before instead of having the initializations scattered across all files and sharing a global parser, what do you think?